roles/postgres/tasks/main.yml

```diff
@@ -42,32 +42,27 @@
     enabled: true
   become: true
 
-- name: Setup postgresql databases
-  community.postgresql.postgresql_db:
-    name: "{{ item.database }}"
-  with_items: "{{ postgres_users }}"
-  become: true
-  become_user: postgres
-
 - name: Setup postgresql users
   community.postgresql.postgresql_user:
-    db: "{{ item.database }}"
     user: "{{ item.username }}"
   with_items: "{{ postgres_users }}"
   become: true
   become_user: postgres
 
-- name: Grant users permissions to create tables in the schema `public`
```

Comment:
Q: Is this gone because users now are not restricted to a certain database?

Reply:
Disregard this comment. I found the answer here -> https://github.com/xsnippet/xsnippet-infra/pull/57/files